app/covidbed/serializer/orga.py

```diff
@@ -0,0 +1,44 @@
+from flask_restful_swagger import swagger
+from flask_restful import fields
+
+
```

Comment:
Suggestion : why not distinguish RequestParam Serialiser and ResponseSerializer ?